packages/commons/src/config-types/workspace-definition/__test__/substituteEnvVariables.test.ts

```diff
@@ -0,0 +1,18 @@
+import yaml from "js-yaml";
+import { substituteEnvVariables } from "../substituteEnvVariables";
+
+describe("substituteEnvVariables", () => {
+    it("basic", () => {
+        process.env.ENV_VAR = "test";
+        const rawYaml = `
+        foo: bar
+        baz: 
+          qux: 
+            thud: \${ENV_VAR}
+        plugh: \${ENV_VAR}`;
+        const parsedYaml = yaml.load(rawYaml);
+        const substitutedYaml = substituteEnvVariables(parsedYaml);
+        console.log(substitutedYaml);
```

Comment:
Prob shouldn’t log in the test

Reply:
ah just did that to get a hold of the object for assertion -- will take out